sdk/test-utils/recorder/vitest.browser.config.ts

```diff
@@ -34,13 +34,14 @@ export default defineConfig({
     watch: false,
     include: ["dist-test/browser/**/*.spec.js"],
     coverage: {
+      enable: true,
```

Comment:
Why aren't we using the base config here?

Reply:
The base depends on test recorder:
https://github.com/Azure/azure-sdk-for-js/blob/c10ccc22d05bf94727be36c5d68e4aa7766442db/vitest.browser.shared.config.ts#L6

Reply:
and this is test recorder